tests/util.rs

```diff
@@ -0,0 +1,401 @@
+#![cfg(feature = "rpc")]
+#![allow(dead_code)]
```

Comment:
is this needed?

Reply:
util is used from the other test rs files. Unfortunately not every test file uses all things in util, so you get dead code even if every single thing is used somewhere. Not sure how to solve this.